src/plugins/nginx.go

```diff
@@ -44,8 +45,9 @@ const (
 var (
 	validationTimeout = 60 * time.Second
 	reloadErrorList   = []*re.Regexp{
-		re.MustCompile(`.*bind\(\) to .* failed \(98: Address already in use\).*`),
-		re.MustCompile(`.*bind\(\) to .* failed \(98: Unknown error\).*`),
+		re.MustCompile(`.*\[emerg\].*`),
+		re.MustCompile(`.*\[alert\].*`),
+		re.MustCompile(`.*\[warn\].*`),
```

Comment:
Maybe remove `re.MustCompile(`.*\[warn\].*`)` from the list and create a separate variable for it like this `warningRegex = re.MustCompile(`.*\[warn\].*`)` since it is handled differently from the other regexes.